code/modules/clothing/factions/elyra.dm

```diff
@@ -0,0 +1,83 @@
+/obj/item/clothing/under/elyra_holo
+	name = "elyran holographic suit"
+	desc = "A marvel of Elyran technology, uses hardlight fabric and masks to transform a skin-tight, cozy suit into cultural apparel of your choosing. Has a dial for Midenean, Aemaqii and Perispolisean clothes respectively."
+	icon = 'icons/clothing/under/uniforms/elyra_holoclothes.dmi'
+	icon_state = "holosuit_fem"
+	item_state = "holosuit_fem"
+	contained_sprite = TRUE
+	action_button_name = "Transform Holoclothing"
+	var/cooldown = 0
+	var/clothing_mode = 0
+	var/list/names = list(
+		"base holographic suit",
+		"medinean artisans holo-dress",
+		"aemaqii ocean holo-dress",
+		"perispolisean desert holo-dress"
+		)
+
+	var/list/descriptions = list(
+		"A form-fitting holographic dress with striking purple and white coloration. It possesses high-fidelity, artistic representation of Elyran Medinean culture markers coating its tightly-woven hardlight fabric.",
+		"A fine holo-dress which owes its overall design to the chemical seas of Aemaqq, with itself flowing gently in waves. It looks particularly cozy, and protective against a cold ocean breeze.",
+		"A loose, airy holo-dress with clashing earthen and bright colors. The design is iconic among the Elyran Perispolisean native populations due to its expressive contrast and the hot climate."
+		)
+
+/obj/item/clothing/under/elyra_holo/masc
+	icon_state = "holosuit_masc"
+	item_state = "holosuit_masc"
+	names = list(
+		"base holographic suit",
+		"medinean artisans holo-suit",
+		"aemaqii fur outfit",
+		"perispolisean nomads outfit"
+		)
+	descriptions = list(
+		"A tight-fitting hardlight suit decorated with golden highlights to contrast its red, purplish overshirt and jet black pants. A favorite among the Elyran planet of Medina.",
+		"High quality, and colorful fur derived from the colder regions of Aemaqq's chemical seas. Were this outfit real, you'd probably assume the thick, stump and even branch-like fur to be extracted from one of its countless sea leviathans. Even the holographic variant is excellent at trapping heat.",
+		"A loose holographic outfit, sturdy, baggy and cool with plentiful pockets. The baggy over-wear is a trend on the Elyran world Perispolis due to its hot climate, and this particular style has spread like wildfire across other warm worlds as well. "
+		)
+
+/obj/item/clothing/under/elyra_holo/attack_self(mob/user)
+	select_appearance(user)
+
+/obj/item/clothing/under/elyra_holo/verb/transform_holoclothing()
+	set name = "Transform Holoclothing"
+	set category = "Object"
+	set src in usr
+	select_appearance(usr)
+
+/obj/item/clothing/under/elyra_holo/proc/select_appearance(mob/user as mob)
+	var/appearance_choice = input("Select an appearance:", "Holoclothing Appearance", names[clothing_mode + 1]) in names
```

Comment:
Make this a radial input menu, please: example of this can be found in appliance or combitool code
Basically make an assoc list that's `name -> image of item`, then do var/thing = RADIAL_INPUT(user, list)

Reply:
Sure, I'll get around to it when I can.